shardingsphere-sql-parser/shardingsphere-sql-parser-dialect/shardingsphere-sql-parser-mysql/src/main/antlr4/imports/mysql/DMLStatement.g4

```diff
@@ -105,11 +105,11 @@ singleTableClause
     ;
 
 multipleTablesClause
-    : tableAliasRefList FROM tableReferences | FROM tableAliasRefList USING tableReferences
+    : multipleTableNames FROM tableReferences | FROM multipleTableNames USING tableReferences
     ;
 
 multipleTableNames
-    : tableName DOT_ASTERISK_? (COMMA_ tableName DOT_ASTERISK_?)*
+    : tableIdentOptWild (COMMA_ tableIdentOptWild)*
     ;
```

Comment:
@lwtdev Maybe change `tableAliasRefList` will be best. and delete unused rule `multipleTableNames`, because the name in MySQL source code is `tableAliasRefList`.
```
tableAliasRefList
    : tableIdentOptWild (COMMA_ tableIdentOptWild)*
    ;
```

Reply:
@jingshanglu  Thanks very much for you advice. I will change it. 
What about `tableLockingList` that references ` tableAliasRefList`. Could I remove it , too? It's unused , but I don't know why this was added.
```
tableLockingList
    : OF tableAliasRefList
    ;
```

Reply:
@jingshanglu I think `tableLockingList` should be add to `lockClause`. but this change need more test case . so I will keep `tableLockingList`  and not modify `lockClause`. what do you think ?

Reply:
> @jingshanglu I think `tableLockingList` should be part of `lockClause`. but this change need more test case . so I will keep `tableLockingList` and not modify `lockClause` this time. what do you think ?

@lwtdev I checked the actual grammar rules([MySQL Source Code](https://github.com/mysql/mysql-server/blob/8.0/sql/sql_yacc.yy)), the `lockClause` rule should like this:
```
lockClause
    : FOR lockStrength tableLockingList? lockedRowAction?
    | LOCK IN SHARE MODE
    ;
```
Change it by the way.

And then can you create PR to master branch?

Reply:
SourceCode:
![image](https://user-images.githubusercontent.com/16559542/98494775-b602f880-2278-11eb-963f-a98aa333ba7a.png)

Reply:
@jingshanglu Hi, do you means repush this pr to master branch or create another pr for the `table_locking_list` issue ?

Reply:
@lwtdev Yes.

Reply:
@lwtdev plz fix the definition of `lockClause` by the way.

Reply:
@jingshanglu  I cannot get your point...
1. fix `lockClause` issue at the same time and repush this pr to master.
2. keep this pr (target branch : apache:5.0.0-alpha-release), then create another pr to fix `lockClause` issue (target branch: master)

Reply:
> 1. fix `lockCl`

@lwtdev 

1. In this PR, fix the definition of `lockClause` at the same time.
2. Create another PR that contains the same content as this PR(target branch:master)

Reply:
@jingshanglu ok